src/helpers/helpers.options.js

```diff
@@ -52,5 +52,5 @@ export function toPosition(value) {
 }
 
 export function isBoundToPoint(options) {
-  return options && (options.xValue || options.yValue);
+  return options && (typeof options.xValue !== 'undefined' || typeof options.xValue !== 'undefined');
```

Comment:
Could use the `defined` helper from Chart.js to make this a bit cleaner.

Reply:
I didn’t know it 😔

Reply:
done